study-builder/studies/testboston/snippets/question-sample-collect-time.conf

```diff
@@ -0,0 +1,27 @@
+{
+  include required("../../snippets/composite-question.conf"),
+  "stableId": ${id.q.sample_collect_time},
+  "allowMultiple": false,
+  "childOrientation": "HORIZONTAL",
```

Comment:
I see. So the reason we split the sample_collect question into two is because we want to layout the time parts horizontally. If we put both date and time into one big composite then we won't be able to lay things out horizontally. I guess this works for now. But the exported data will look wonky and it will show up weird in DSM's data browser... I guess that's the trade-off for now.

Reply:
Tried as one composite question but because of layout issue had to split it.